projects/schematics/src/migrations/2_0/constructor-deprecations/constructor-deprecation-data.ts

```diff
@@ -18,11 +18,16 @@ import { CHECKOUT_PAYMENT_SERVICE_MIGRATION } from './data/checkout-payment.serv
 import { CHECKOUT_PROGRESS_MOBILE_TOP_COMPONENT_MIGRATION } from './data/checkout-progress-mobile-top.migration';
 import { CHECKOUT_GUARD_MIGRATIONS } from './data/checkout.guard.migration';
 import { CHECKOUT_SERVICE_MIGRATION } from './data/checkout.service.migration';
+import { CLIENT_AUTHENTICATION_TOKEN_SERVICE_MIGRATION } from './data/client-authentication-token.service.migration';
 import { CONFIGURABLE_ROUTES_SERVICE_MIGRATION } from './data/configurable-routes.service.migration';
 import { CURRENT_PRODUCT_SERVICE_MIGRATION } from './data/current-product-service.migration';
 import { EXTERNAL_ROUTES_SERVICE_MIGRATION } from './data/external-routes.service.migration';
+import { LOGIN_FORM_COMPONENT_MIGRATION } from './data/login-form.component.migration';
```

Comment:
this migration is not part of this PR

Reply:
yup, but I believe it was just a change of position from prettier since I don't use those migration in my PR

Reply:
got moved from line 46